pom.xml

```diff
@@ -31,6 +31,12 @@
   <dependencyManagement>
     <dependencies>
       <dependency>
+        <groupId>org.jenkins-ci.main</groupId>
+        <artifactId>jenkins-core</artifactId>
+        <version>1.430</version>
```

Comment:
Why 1.430? It's six years old…

Reply:
Why not? The tendency is not to use the lower required :smile:

Reply:
@daniel-beck Increasing the version implies a lot of code changes (and not straightforward one). Probably it needs to be done, but it's not what this PR is trying to fix.

Reply:
@daniel-beck, @varyvol is right. This PR is trying to align the jenkins-core version in both modules, `plugins-compat-tester` and `plungins-compact-tester-model`. My intention is just to fix both core version and version 1.430 is the one used by `plugins-compat-tester` (higher than the one used by `plungins-compact-tester-model`)